features/optional-catch-binding.yml

```
@@ -0,0 +1,7 @@
+name: Optional catch binding
+description: Omit the the binding parameter of a `catch` clause when you don't need information about the exception in a `try...catch` statement.
```

We've offset `…` as a placeholder with spaces elsewhere 

```suggestion
description: Omit the the binding parameter of a `catch` clause when you don't need information about the exception in a `try ... catch` statement.
```